provider/lxd/environ.go

```diff
@@ -104,11 +105,36 @@ func (env *environ) profileName() string {
 	return "juju-" + env.Name()
 }
 
+type LXDProfiler interface {
```

Comment:
I think this should be in the [environs](https://github.com/juju/juju/blob/develop/environs/interface.go) package?